src/cpp/client.cpp

```diff
@@ -702,25 +696,15 @@ parsed_reply_map Client::get_db_cluster_info(std::string address)
     if (host.empty() or port == 0)
         throw std::runtime_error(std::string(address) +
                                  "is not a valid database node address.");
-    Command cmd;
+    ClusterInfoCommand cmd;
     cmd.set_exec_address_port(host, port);
     cmd.add_field("CLUSTER");
     cmd.add_field("INFO");
     CommandReply reply = this->_run(cmd);
-    return CommandReplyParser::parse_db_cluster_info(std::string(reply.str(),
+    return cmd.parse_db_cluster_info(std::string(reply.str(),
```

Comment:
Since ``parse_db_cluster_info`` is static should we call it like others: ``ClusterInfoCommand::parse_db_cluster_info(reply)``?